library/geonetwork

```diff
@@ -12,9 +12,17 @@ GitCommit: e5cf5da76f557481c35b4a4d3e3cac77768a1302
 Directory: 3.0.5/postgres
 
 Tags: 3.2.2, 3.2, latest
-GitCommit: 7ba5f533d9ee31cf1a22d4d6a1a84fbb74f86466
+GitCommit: a9d9f3f07c0f2b8badadb74206647d7f6dea4b24
 Directory: 3.2.2
 
 Tags: 3.2.2-postgres, 3.2-postgres, postgres
 GitCommit: 7ba5f533d9ee31cf1a22d4d6a1a84fbb74f86466
 Directory: 3.2.2/postgres
+
+Tags: 3.4.0
```

Comment:
Did you want to move `latest` to `3.4.0` and have a `3.4` alias (and move `postgres` and add `3.4-postgres` below)?

Reply:
Yes, thanks for pointing that out. I have fixed that in my latest [commit](https://github.com/docker-library/official-images/pull/3792/commits/fb6c4bfddb5a131017e0fa2f0cd732c0ea25514d).